services/queue/test/helper.js

```diff
@@ -63,7 +63,11 @@ exports.withS3 = (mock, skipping) => {
       exports.load.cfg('aws.accessKeyId', undefined);
       exports.load.cfg('aws.secretAccessKey', undefined);
 
-      const mock = new mockAwsS3.S3({});
+      const mock = new mockAwsS3.S3({
+        params: {
```

Comment:
this is needed for mockAwsS3 to properly construct temp folder path.
Without it, some calls are working fine (the one which contain `Bucket` key, and the ones that don't - fail)